### PR TITLE
Create prototype of OSAL component

### PR DESCRIPTION
**Pull request by Bzykowa**

### CODE REVIEW

File: client/osal/__init__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+"""Operating System Abstraction Layer"""
```
Comment (aCinal) [inline client/osal/__init__.py:1]: Please install pre-commit hooks as described in README.md:
> pip install pre-commit
> pre-commit install
and clean up the code locally

File: client/osal/__init__.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    def hardware_fingerprint(self):
+        """ HWF = UUID of a drive + model of CPU + username of os user account -> sha256 hash """
+        try:
+            drive_UUID = subprocess.check_output(
+                'blkid -s UUID -o value | head -n 1')
+            cpu_model = subprocess.check_output(
+                'lscpu | grep \'Model name\' | cut -f 2 -d ":" | awk \'{$1=$1}1\'')
+            username = getpass.getuser()
+        except:
+            """ If in docker container and no devices present use default placeholder vals """
```
Comment (aCinal) [inline client/osal/__init__.py:42]: Please use """(...)""" comments for docstrings only and for inline comments use `#`

Comment (Bzykowa) [reply client/osal/__init__.py]: I changed that line to inline comment (:

File: client/osal/__init__.py (hunk 1 above)
Comment (Bzykowa) [reply client/osal/__init__.py]: Hooks installed 👍

**Review by aCinal — APPROVED** [review]

**Pull request merged**